documentation/book/proc-replacing-private-keys.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='proc-replacing-private-keys-{context}']
+
+= Replacing private keys
+
+You can replace the private keys used by the cluster CA and clients CA certificates. When a private key is replaced, the Cluster Operator generates a new CA certificate for the private key.
+
+.Prerequisites
+
+* The Cluster Operator is running.
+* A Kafka cluster in which CA certificates and private keys are installed.
+
+.Procedure
+
+. View a description of the private key `Secret` that contains the private key that you want to renew.
```

Comment:
Why is the user supposed to view the Secret? Seem unnecessary / this step doesn't explain it.

Reply:
I wanted to help the the user to annotate the correct Secret. But on second reading it does seem like repetition. I'll remove step one from both procedures.